src/safeds/data/tabular/transformation/_functional_table_transformer.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from safeds._utils import _structural_hash
+
+if TYPE_CHECKING:
+    from collections.abc import Callable
+
+    from safeds.data.tabular.containers import Table
+
+from ._table_transformer import TableTransformer
+
+
+class FunctionalTableTransformer(TableTransformer):
+    """
+    Learn a transformation for a set of columns in a `Table` and transform another `Table` with the same columns.
+
+    Parameters
+    ----------
+    funct:
+        The Callable that receives a table and returns a table that is to be wrapped.
+    """
+
+    # ------------------------------------------------------------------------------------------------------------------
+    # Dunder methods
+    # ------------------------------------------------------------------------------------------------------------------
+
+    def __init__(self,
+                 funct: Callable[[Table], Table],
```

Comment:
```suggestion
                 transformer: Callable[[Table], Table],
```

See issue. Use a rename refactoring. Don't forget to update the docstring.

Reply:
Done